index.d.ts

```diff
@@ -120,12 +522,37 @@ declare module 'evergreen-ui' {
   export class Card extends React.PureComponent<CardProps> {
   }
 
+  export interface CodeProps extends TextProps<'code'> {
+
+  }
```

Comment:
Might be nicer to use `type CodeProps = TextProps<'code'>`?

Reply:
Ok!